src/main/java/com/redhat/devtools/intellij/common/actions/AbstractActionVisibility.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
```

Comment:
2021

Reply:
file included by mistake. Shouldn't have made it into this commit.